pkgs/tools/misc/fclones/default.nix

```diff
@@ -19,7 +19,7 @@ rustPlatform.buildRustPackage rec {
   cargoHash = "sha256-C7DKwEMYdypfItflMOL7rjbAdXDRsXDNoPlc9j6aBRA=";
 
   buildInputs = lib.optionals stdenv.isDarwin [
-    darwin.apple_sdk_11_0.frameworks.AppKit
+    darwin.apple_sdk_11_1.frameworks.AppKit
```

Comment:
Perhaps add a `apple_sdk_11` or even just `apple_sdk_new`. What the vast majority of these packages actually want is a non-ancient SDK, not 11.0 or 11.1 specifically.

Reply:
Proposed `darwin.apple_sdk_11` in #234763, but lost interest as there were no feedbacks...